### PR TITLE
Project proposal for MadAnalysis 5 (2022)

### PR DESCRIPTION
**Pull request by jackaraz**
This PR consists of project proposal for Google Summer of Code 2022 based on [MadAnalysis 5](http://madanalysis.irmp.ucl.ac.be) software. We added the project proposal, description for the organizations that are involved and description for the tool. All corresponding logos are added to the `images` folder.

Please let us know if there is anything missing in our proposal.

PS: @agheata @aflp91 @hegner the [website](https://hepsoftwarefoundation.org/gsoc/guideline.html) requires me to add one of you as a reviewer but I can not do it since I don't have the permissions for the repository.

Thanks

### CODE REVIEW

File: _gsocorgs/2022/iphc.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "IPHC"
+author: "Eric Conte"
+layout: default
+organization: IPHC
+logo: IPHC-logo.png
+description: |
+    [IPHC](http://www.iphc.cnrs.fr) Founded in 2006, the IPHC ("Institut Pluridisciplinaire Hubert Curien" for "Hubert Curien Multidisciplinary Institute") is an international renown laboratory located in Strasbourg (France). Its specificity is to gather three department of different sciences: eco-physiology, chemistry and subatomic physics. The package MadAnalysis 5 was in particular designed in the latter department for the needs of phenomenology in particle physics at collider. The IPHC is a member institute of several international experimental collaborations such as CMS, ALICE, FCC, ILC, Juno, LIGO-Virgo, Antares-KM3NET, …
```
Comment (hegner) [inline _gsocorgs/2022/iphc.md:8]: department -> departments

**Review by hegner — CHANGES_REQUESTED** [review]
Just a minor typo. Otherwise OK for addition. Thanks!

File: gsoc/2022/mentors.md [hunk 2]
```diff
@@ -15,3 +15,6 @@ layout: plain
 * Javier Lopez-Gomez [Javier Lopez-Gomez](mailto:j.lopez@cern.ch) CERN
 * Dalila Salamani [Dalila Salamani](mailto:dalila.salamani@cern.ch) CERN
 * Anna Zaborowska[Anna Zaborowska](mailto:anna.zaborowska@cern.ch) CERN
+* Benjamin Fuks [fuks@lpthe.jussieu.fr](mailto:fuks@lpthe.jussieu.fr) LPTHE
```
Comment (hegner) [inline gsoc/2022/mentors.md:18]: sorry, did not spot this earlier. Could you try to put your names in the alphabetical order and insert them appropriately?

Comment (jackaraz) [reply gsoc/2022/mentors.md]: updated accordingly

**Pull request merged**